### PR TITLE
remove double dash

### PR DESCRIPTION
**Pull request by marekbiskup**

### CODE REVIEW

File: docs/design/networking.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ With the IP-per-pod model, all user containers within a pod behave as if they ar
...
+The approach does reduce isolation between containers within a pod - ports could conflict, and there couldn't be private ports across containers within a pod, but applications requiring their own port spaces could just run as separate pods and processes requiring private communication could run within the same container. Besides, the premise of pods is that containers within a pod share some resources (volumes, cpu, ram, etc.) and therefore expect and tolerate reduced isolation. Additionally, the user can control what containers belong to the same pod whereas, in general, they don't control what pods land together on a host.
```
Comment (erictune) [inline docs/design/networking.md]: The [correct punctuation](http://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0002.html) here is an em dash.  Suggest changing `--` to `&mdash;`, which will render nicely &mdash; like that.

Comment (marekbiskup) [inline docs/design/networking.md]: Good point. I wasn't aware of that

**Comment by marekbiskup** [comment]
@erictune , please check if this is better.
I made this change only in these three files.

**Comment by erictune** [comment]
lgtm

**Pull request merged**